sklearn/utils/fixes.py

```diff
@@ -24,11 +24,18 @@
 
 from .deprecation import deprecated
 
-try:
-    from pkg_resources import parse_version  # type: ignore
-except ImportError:
-    # setuptools not installed
-    parse_version = LooseVersion  # type: ignore
+
+def parse_version(v):
```

Comment:
We can have a small test to make sure that `parse_version` works versions with dev version numbers.

Reply:
sure